src/store/feed/actions.js

```diff
@@ -173,3 +174,11 @@ export const seenNotifications = arr => async (dispatch, getState) => {
   const results = await Kitsu.one('activityGroups', id).all('_seen').post(arr);
   console.log(results);
 };
+
+export const markNotifications = (id, token, notifs) => {
+  fetch(`${kitsuConfig.baseUrl}/edge/feeds/notifications/${id}/_read`, {
```

Comment:
We should probably use the Kitsu API client wherever we're doing API access for consistency. I think it's in `kitsu/config/api`, which is an odd place for it, but that's where it is at the moment.

Reply:
this one's a custom endpoint that doesn't match jsonapi conventions, so there isn't actually a way to use it via our jsonapi client, afaik. attempted with the url builder, but it wants definitions, which we can't provide since this doesn't follow the standard for model endpoints.

Reply:
Fair enough! In that case ignore this feedback.